app/css/components/markdown-editor.css

```diff
@@ -25,7 +25,12 @@
                 @apply hidden;
             }
         }
+
+        & .CodeMirror-line {
+            visibility: hidden;
+        }
```

Comment:
What's this do?

Reply:
@iHiD If a user types in the textbox and hits cancel, the text they typed goes over the "Add a comment" pseudo element you added. So I just hide the text they typed in when the editor is compressed.

Reply:
Thanks. I'll check this out to understand it better :)

Reply:
> If a user types in the textbox and hits cancel, the text they typed goes over the "Add a comment" pseudo element you added. So I just hide the text they typed in when the editor is compressed.

I don't understand this. Someone can't press cancel until the text box is empty, so how are they able to still have text in the text box and press cancel?

Reply:
It is because we have localstorage. Refreshing the page has the saved content from localstorage on the compressed editor.